schema/dataKeys.json

```diff
@@ -0,0 +1,286 @@
+        },
+        "pipelineImage": {
+          "type": "string",
+          "description": ""
+        }
+      }
+    },
+    "github": {
+      "type": "object",
+      "properties": {
+        "githubSecret": {
+          "type": "string",
+          "description": "The secret used to authenticate GitHub via the the GitHub token"
+        }
+      }
+    },
+    "pyxis": {
+      "type": "object",
+      "properties": {
+        "secret": {
+          "type": "string",
+          "description": "The secret used to authenticate Pyxis e.g. example-collect-pyxis-params-cert"
+        },
+        "server": {
+          "type": "string",
+          "description": "The Pyxis server being used e.g. production-internal"
+        },
+        "skipRepoPublishing": {
+          "type": "boolean",
+          "description": "Skip repo publishing"
+        }
+      }
+    },
+    "application": {
+      "type": "string",
+      "description": "The application name e.g. example-app"
+    },
+    "slack": {
+      "type": "object",
+      "properties": {
+        "slack-notification-secret": {
+          "type": "string",
+          "description": "The secret key for slack notifications e.g. example-team-slack-webhook-notification-secret"
+        },
+        "slack-webhook-notification-secret-keyname": {
+          "type": "string",
+          "description": "The key name for the slack webhook notification secret e.g. release"
+        }
+      }
+    },
+    "infra-deployment-update-script": {
+      "type": "string",
+      "description": "A script that can alter files before a PR is created"
+    },
+    "mapping": {
+      "type": "object",
+      "properties": {
+        "components": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "description": "The component name found in the snapshot e.g. example-component"
+              },
+              "repository": {
+                "type": "string",
+                "description": "Url where you want to push the artifact e.g. quay.io/redhat/example-component"
+              },
+              "tags": {
+                "type": "array",
+                "description": "The tags to push the artifact e.g. {{ git_sha }},{{ digest_sha }}, 1.0",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "pushSourceContainer": {
+                "type": "boolean",
+                "description": "Indicates if the source container should be pushed"
+              }
+            }
+          }
+        },
+        "defaults": {
+          "type": "object",
+          "properties": {
+            "tags": {
+              "type": "array",
+              "description": "The default tags to push for all components e.g. {{ git_sha }},{{ digest_sha }}, 1.0",
```

Comment:
It being an array, I think you should add `[]` before and after in the e.g.

Reply:
fixed